src/shmem_compiler_script.in

```diff
@@ -118,6 +119,7 @@ my $include_flag = "-I";
 my $libdir_flag = "-L";
 
 my $comp = "";
+my $wrapper = (split(/\//, $wrapperdir))[-1];
```

Comment:
Why split instead of basename?  Basename should be more robust; I might be able to break this split with spaces or other characters in a directory/filename.